agents/community/aider/src/aider_agent/agent.py

```diff
@@ -5,23 +5,145 @@
 import sys
 import tempfile
 from pathlib import Path
+from collections.abc import AsyncGenerator
+from textwrap import dedent
 
-from pydantic import Field
-from beeai_sdk.schemas.text import TextInput, TextOutput
-from beeai_sdk.providers.agent import Server, Context
+from acp_sdk import Message, Metadata, Link, LinkType, MessagePart, Artifact
+from acp_sdk.server import Context, Server
+from pydantic import AnyUrl
+import mimetypes
 
+server = Server()
 
-class Output(TextOutput):
-    files: dict[str, str] = Field(default_factory=dict)
-    text: str = Field(default_factory=str)
 
+@server.agent(
+    name="aider",
+    description="An AI pair programmer that edits code in a local Git repository using natural language, executing commands and providing feedback.",
```

Comment:
ultra nitpick: I'm using the function name and docstring in other agent, do we want to stay consistent?